components/messageBox/messageBox.js

```diff
@@ -0,0 +1,77 @@
+import React, { PropTypes } from 'react';
```

Comment:
please import PropTypes from `prop-types` package. otherwise we keep adding more to our debt when upgrading to React v15+.